src/OrchardCore/OrchardCore.DisplayManagement/Zones/ZoneShapes.cs

```diff
@@ -105,7 +105,7 @@ public async Task<IHtmlContent> ContentZone(dynamic DisplayAsync, dynamic Shape,
             }
             else
```

Comment:
can you make it `else if (groupings.Count == 1)` here, rather than another if block inside it

Reply:
Oh, of course.Done.